src/behavior-considered-undefined.md

```diff
@@ -23,7 +23,8 @@ code.
 </div>
 
 * Data races.
-* Dereferencing (using the `*` operator on) a dangling or unaligned raw pointer.
+* Evaluating a dereference [place expression] (`*expr`) on a raw pointer that is
+  [dangling] or unaligned.
```

Comment:
I think this was raised in the Zulip conversation, but I'll nevertheless raise it here: I wonder if a word other than e-_valu_-ating could be used when talking about a _place_ expression (although I couldn't come up with one satisfactory enough, so feel free to disregard this nit).

Reply:
Clearly the right term is "placating".

On a more serious note, this is one reason why I argued fiercly against calling these things "values", but I was unable to convince enough people to make a difference...